lib/modules/platform/gitea/index.md

```diff
@@ -18,7 +18,10 @@ You must set `platform=gitea` in your Renovate config file.
 The PAT should have these permissions:
 
 - `repo`
-- `read:user` and `read:email`
+- `read:user`
+- `read:misc` (since Gitea version 1.20.0)
```

Comment:
no longer required since v1.20.2

Reply:
Ok, I updated it in https://github.com/renovatebot/renovate/pull/23615/commits/17e79c0190d361b19e7edd51eb9303573892af29 .